docs/Installation.md

```diff
@@ -90,6 +90,6 @@ Uninstallation is documented in the [FAQ](FAQ.md).
 
 <a name="2"><sup>2</sup></a> macOS 11 (Big Sur) or higher is best and supported, 10.11 (El Capitan) – 10.15 (Catalina) are unsupported but may work and 10.10 (Yosemite) and older will not run Homebrew at all. For 10.4 (Tiger) – 10.6 (Snow Leopard) see [Tigerbrew](https://github.com/mistydemeo/tigerbrew).
 
-<a name="3"><sup>3</sup></a> You may need to install Xcode, the CLT, or both depending on the formula, to install a bottle (binary package) which is the only supported configuration. Downloading Xcode may require an Apple Developer account on older versions of Mac OS X. Sign up for free at [Apple's website](https://developer.apple.com/register/index.action).
+<a name="3"><sup>3</sup></a> You may need to install Xcode, the CLT, or both depending on the formula, to install a bottle (binary package) which is the only supported configuration. Downloading Xcode may require an Apple Developer account on older versions of macOS. Sign up for free at [Apple's website](https://developer.apple.com/account/).
```

Comment:
@gerlero @carlocab sorry, this is wrong: it should be `Mac OS X` because it only applies there!

Reply:
https://github.com/Homebrew/brew/pull/15700

Reply:
Well, I got it wrong then (honestly, I'm not really sure how far back one can go before Homebrew doesn't install anymore). Thanks for the check!